eng/common/scripts/Verify-Links.ps1

```diff
@@ -188,7 +188,11 @@ function ParseLinks([string]$baseUri, [string]$htmlContent)
 
   #$links | Foreach-Object { Write-Host $_ }
 
-  return $links
+  if ($null -eq $links) {
+    $links = @()
+  }
+
+  return ,$links
```

Comment:
FYI without the `,` in `,$links`, the `@()` would be converted to `$null`:

https://stackoverflow.com/a/18477004/986533